azurepipelines/e2e_test/e2etest.yaml

```diff
@@ -214,51 +244,54 @@ stages:
-                          chmod u=rwx,g=rwx,o=rx ./testsetup/setup.sh &&
-                          ./testsetup/setup.sh --distro ubuntu-20.04 --architecture arm64
+                          chmod u=rwx,g=rwx,o=rx ./testsetup/sas_vm_setup.sh &&
+                          ./testsetup/sas_vm_setup.sh --distro ubuntu-20.04 --architecture arm64
                       image_working_directory: $(E2E_WORKING_DIR)
-          # - job: SetupUbuntu2204_AMD64_VM
-          #   displayName: "Setting up VM for Ubuntu 22.04 AMD64 Test VM"
-          #   timeoutInMinutes: 360
-          #   cancelTimeoutInMinutes: 360
-          #   pool: aduc_1es_client_pool
-          #   steps:
-          #       - template: ./templates/e2e_vm_setup.yaml
-          #         parameters:
-          #             distroName: ubuntu-22.04-amd64
-          #             vm_size: Standard_DS1_v2
-          #             image_publisher: canonical
-          #             image_offer: 0001-com-ubuntu-server-focal
-          #             image_sku: 22_04-lts
-          #             image_version: latest
-          #             packagePattern: "*ubuntu2204-amd64*/*.deb"
-          #             package_pipeline: "Azure.adu-private-preview.e2e-test"
-          #             du_tarball_script: >-
-          #                 tar -xf /tmp/testsetup.tar.gz -C ./ &&
-          #                 chmod u=rwx,g=rwx,o=rx ./testsetup/setup.sh &&
-          #                 ./testsetup/setup.sh --distro ubuntu-22.04 --architecture amd64
-          #             image_working_directory: $(E2E_WORKING_DIR)
-          # - job: SetupUbuntu2204_ARM64_VM
-            # displayName: "Setting up VM for Ubuntu 22.04 ARM64 Test VM"
-            # timeoutInMinutes: 360
-            # cancelTimeoutInMinutes: 360
-            # pool: aduc_1es_client_pool
-            # steps:
-            #     - template: ./templates/e2e_vm_setup.yaml
-            #       parameters:
-            #           distroName: ubuntu-22.04-arm64
-            #           vm_size: Standard_D2plds_v5
-            #           image_publisher: canonical
-            #           image_offer: 0001-com-ubuntu-server-focal
-            #           image_sku: 22_04-lts-arm64
-            #           image_version: latest
-            #           packagePattern: "*ubuntu2204-arm64*/*.deb"
-            #           package_pipeline: "Azure.adu-private-preview.e2e-test"
-            #           du_tarball_script: >-
-            #               tar -xf /tmp/testsetup.tar.gz -C ./ &&
-            #               chmod u=rwx,g=rwx,o=rx ./testsetup/setup.sh &&
-            #               ./testsetup/setup.sh --distro ubuntu-22.04 --architecture arm64
-            #           image_working_directory: $(E2E_WORKING_DIR)
+                      device_id_under_test: "ubuntu-20.04-arm64-sas"
+        #   # - job: SetupUbuntu2204_AMD64_VM
+        #   #   displayName: "Setting up VM for Ubuntu 22.04 AMD64 Test VM"
```

Comment:
Why is this still commented out? Is there a build breaking issue we need to address?

Reply:
it will be on a separate pr